.github/ISSUE_TEMPLATE/feature-spec.md

```diff
@@ -21,3 +21,18 @@ _What metrics will be used to measure impact?_
 
 ## Documentation 
 _The goal of the documentation should be to make it easier for others to do this_
+
+<details>
+  <summary>Triage Template (team only)</summary>
```

Comment:
Maybe use the term `core team` as the word `team` is also used to refer to the other Github teams such as recurring contributors etc? 🙂 

```suggestion
  <summary>Triage Template (core team only)</summary>
```